WordPress/gutenberg
# Remove OffCanvasEditor

## What?
This removes the OffCanvasEditor component.

## Why?
This component isn't used anymore.

## How?
Remove the code and anywhere that references it.

## Testing Instructions
Check that the List View inside the Navigation block and the Navigation section of Browse Mode still work as expected.
